checkstyle-custom/src/main/java/io/vivy/checkstyle/custom/NoLineBreakAfterAssignCheck.java

```diff
@@ -0,0 +1,47 @@
+package io.vivy.checkstyle.custom;
+
+import com.puppycrawl.tools.checkstyle.api.AbstractCheck;
+import com.puppycrawl.tools.checkstyle.api.DetailAST;
+import com.puppycrawl.tools.checkstyle.api.TokenTypes;
+import com.puppycrawl.tools.checkstyle.utils.CommonUtil;
+
+/**
+ * This check ensures that there is no line break immediately after an assign literal.
+ */
+public class NoLineBreakAfterAssignCheck extends AbstractCheck {
+
+    private static final String MSG_KEY = "Line break immediately after assign literal is not allowed";
+
+    @Override
+    public int[] getDefaultTokens() {
+        return getAcceptableTokens();
+    }
+
+    @Override
+    public int[] getAcceptableTokens() {
+        return new int[]{TokenTypes.ASSIGN, TokenTypes.PLUS_ASSIGN};
+    }
+
+    @Override
+    public int[] getRequiredTokens() {
+        return CommonUtil.EMPTY_INT_ARRAY;
+    }
+
+    @Override
+    public void visitToken(DetailAST ast) {
+
```

Comment:
so, is there any rule that restricts blank lines after method signature?

Reply:
I will add that